plugins/put-a-file-with-ftp/impl/src/main/java/org/pentaho/di/job/entries/ftpput/JobEntryFTPPUT.java

```diff
@@ -642,13 +642,18 @@ public Result execute( Result previousResult, int nr ) {
         }
       }
 
-      Path path = Paths.get( realLocalDirectory );
-      if ( !Files.isReadable( path ) || !Files.isWritable( path ) ) {
-        throw new FTPException( BaseMessages.getString( PKG, "JobFTPPUT.LocalDir.NoPermission" ) );
+      boolean isStandardFileSystem = (!realLocalDirectory.contains("://")) || realLocalDirectory.equalsIgnoreCase("file");
+      if (isStandardFileSystem) {
+        if (realLocalDirectory.charAt(0) == '/')
```

Comment:
please enclose with braces even for a single line of code suggested by sonarqube rules